delta/plugins/graph-analytics/src/main/resources/elasticsearch/mappings.json

```diff
@@ -9,6 +9,38 @@
     "_rev": {
       "type": "long"
     },
+    "_createdAt": {
```

Comment:
I added those at it is an trivial addition and it can help to solve some questions asked in past